GoteoFoundation/goteo
# Home values module

Add values module at home

### Files
HTML: Resources/templates/responsive/home/partials/values.php
CSS: public/assets/sass/layouts/_home.scss
JS: public/assets/js/home/home.js
ICONS: public/assets/img/home

### Testing
1. Check if elements are well placed in desktop and mobile
2. Check if sliders are working
3. Check if project images are tinted
4. Check if footprints tab are working
